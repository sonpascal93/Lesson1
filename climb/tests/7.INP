200
1 -12 49 19 49 -13 -87 91 89 79 -59 -45 -47 14 -25 -81 0 38 85 53 -87 -44 -91 88 -8 57 -13 20 -97 -53 -39 29 -48 -45 -6 0 -11 -21 -81 57 -75 0 -23 5 85 43 25 -68 96 -17 -93 -84 -73 -98 -87 93 98 -11 -23 -35 39 71 41 27 12 -81 92 -50 -73 28 -32 -89 37 -70 -44 67 0 30 -16 -58 -12 -2 -84 38 -27 -9 11 24 0 -98 82 85 -58 87 74 -68 -96 -56 17 -39 -68 13 14 3 -43 23 97 28 -54 52 72 -58 -44 53 0 -74 10 -51 -32 -53 6 -44 84 24 77 -70 0 -19 -20 -84 -43 51 -83 -44 53 -63 -9 70 6 98 -29 35 -48 -64 -37 79 56 0 65 -16 33 -73 24 84 16 58 0 -75 34 -32 -89 71 82 -16 -86 2 92 18 68 -75 -75 75 -24 -61 48 20 53 71 24 9 52 42 51 55 11 -43 -39 74 -19 38 73 12 90 32 -92 -20 93 54 24 31 