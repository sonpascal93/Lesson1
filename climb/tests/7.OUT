34
7 21 30 33 34 35 36 42 44 47 64 70 78 84 91 92 94 95 99 102 104 115 127 128 129 131 134 136 144 152 158 170 171 195 